index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>ScriptTree</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <div class="header">
+        <h1>Why Kill a Tree on Christmas?</h1>
+    </div>
+    <div class='workspc'>
+        <form action='' name='theForm' method="get">
+            <div class='formSpc1'>
+                <label>Please introduce Pine Height  <input type="number" name="height" maxlength="2" id='heightNumber'></label>
+            </div>
+            <br><br>
+            <div class='formSpc'>
+                <label>Please introduce character to grow the Pine  <input type="text" name="character" maxlength="1" id='charType'></label>
+            </div>
+<br><br><br>
+            <input type="button" name="treebutton" value="Grow Your Tree" id='fertilizer'>
+        </form>
+    </div>
+<script type="text/javascript" src='script.js'></script>
+</body>
+</html>
```

Comment:
Always add a new line to your files. It's best to make sure your editor does this automatically.